### ISSUE TITLE
TypeError: this.resourceListURL is not a function

### ISSUE THREAD

**Issue by benjaminapetersen** [openshift/origin-web-console#1393]
While on this page: `https://localhost:9000/dev-console/project/my-project-2/browse/dc/dctest?tab=history` I was attempting to delete a DC which triggered this error:

![screen shot 2017-04-03 at 2 16 12 pm 1](https://cloud.githubusercontent.com/assets/280512/24623977/4bbd7012-1878-11e7-82a0-a74f9e4dd6ba.png)

Looks like an extraction issue w/pulling things out into `origin-web-common`. 

@spadgett @jwforres

**Comment by jwforres** [comment]
@jeff-phillips-18 also something you could look at, related to delete directive moving to common probably

**Comment by spadgett** [comment]
We're referencing `resourceListURL` in common, but it lives in origin-web-console.

```
➜  origin-web-common git:(master) rg resourceListURL
src/components/delete-link/deleteLink.js
105:            $location.url(this.resourceListURL(APIService.kindToResource(scope.kind), scope.projectName));
```

I'd rather not push that function into common since it's very specific to the web console routes.

**Comment by jeff-phillips-18** [comment]
@jwforres yeah, I'll take this one

**Comment by jeff-phillips-18** [comment]
@spadgett  yes, this is issue https://github.com/openshift/origin-web-common/issues/9

**Comment by spadgett** [comment]
Fixed by #1404
